src/App.js

```diff
@@ -4,13 +4,17 @@ import Navbar from "./Components/Navbar/Navbar";
 import {Route,Routes} from "react-router-dom"
 import Home from "./Pages/Home/Home";
 import Mockman from "mockman-js";
+import { Login } from "./Pages/Authentication/Login";
+import Signup from "./Pages/Authentication/Signup";
 
 function App() {
   return (
     <div className="App">
       <Navbar/>
       <Routes>
         <Route path="/" element={<Home/>}  />
+        <Route path="/login" element={<Login/>}  />
+        <Route path="/signup" element={<Signup/>}  />
         <Route path="mockman" element={<Mockman/>} />
```

Comment:
1. You can add a 404 page, if the page is not in the routes refer  this(https://ui.dev/react-router-handling-404-pages)